packages/kit/src/runtime/control.js

```diff
@@ -31,3 +31,28 @@ export class Redirect {
 		this.location = location;
 	}
 }
+
+export class ValidationError {
+	/**
+	 * @param {number} status
+	 * @param {Record<string, any>} errors
+	 * @param {FormData | Record<string, any> | null | undefined} [values]
+	 */
+	constructor(status, errors, values) {
```

Comment:
I was thinking `(status, fields, errors)` would be preferable in the common case where you're just passing `fields` back, because this...

```js
throw invalid(400, fields, {
  password: 'Your password is too secure'
});
```

is more natural than

```js
throw invalid(400, {
  password: 'Your password is too secure'
}, fields);
```

which Prettier would turn into this monstrosity:

```js
throw invalid(
  400,
  {
    password: "Your password is too secure",
  },
  fields
);
```

Reply:
ok, switching it